tests/product/resources/connector_permissions_warning.txt

```diff
@@ -1,24 +1,28 @@
 
-Warning: [master] Error validating /etc/opt/prestoadmin/connectors/tpch.properties
+Fatal error: [%(slave3)s] Error validating /etc/opt/prestoadmin/connectors/tpch.properties
```

Comment:
We had spoken about not hard coding the hostnames for the docker cluster so that you can run more than one test at a time, so I didn't want to use hardcoded names for the things I was changing.